README.md

```diff
@@ -33,6 +34,7 @@ var MyComponent = React.createClass({
           <SomeOtherAmazingComponent />
```

Comment:
I think you need to split the template string like the following in order to render `<SomeOtherAmazingComponent />` as a sub-component:

```jsx

{`
  ## Reasons React is great

  1. Server-side rendering
  2. This totally works:
`}

<SomeOtherAmazingComponent />

{`
  Pretty neat!
`}
```